CONTRIBUTING

```diff
@@ -20,25 +19,22 @@ the directory 'Documentation' of the Linux kernel source tree.
 
 It can be accessed online too:
 
-http://lxr.linux.no/linux+v2.6.27/Documentation/CodingStyle
+https://www.kernel.org/doc/html/v4.10/process/coding-style.html
 
 Submitting Modifications:
 -------------------------
 
-The contributions should be email patches or github pull requests.
-The guidelines are the same as the patch submission for the Linux kernel
-except for the DCO which is defined below. The guidelines are defined in the
-'SubmittingPatches' file, available in the directory 'Documentation'
-of the Linux kernel source tree.
-
-It can be accessed online too:
-
-https://www.kernel.org/doc/Documentation/SubmittingPatches
-
-You can submit your patches to the lxc-devel@lists.linuxcontainers.org mailing
-list. Use http://lists.linuxcontainers.org/listinfo/lxc-devel to subscribe
-to the list.
-
+The contributions must be Github pull requests.
```

Comment:
This contradicts itself in the next line. You can't say "must be Github pull requests" and then say that e-mail works too :)

The reality is that we prefer Github PRs over e-mail.